CITATION.cff

```diff
@@ -76,9 +76,9 @@ authors:
   family-names: Wessel
   affiliation: University of Hawaiʻi at Mānoa, USA
   orcid: https://orcid.org/0000-0001-5708-7336
```

Comment:
Didn't want to bring this up so soon, but since Paul passed away, I'm wondering what the convention was in keeping his name on the citation list? Do we remove his name now, keep his name on for a couple more releases, or remove it after his funding/grants run out? See e.g. https://academia.stackexchange.com/questions/116845/dead-author-ethical-guidelines

Reply:
Personally, I prefer to keep Paul's name at least in the period of the funding.